### DISCUSSION TITLE
Ability to set permanent password during GitHub build actions

### DISCUSSION THREAD

**Discussion by kb9gxk** [rustdesk/rustdesk#4646] [Feature Request]
I'm creating builds for my job that will get installed on all the computers to support my users.  It would be nice to preset the permanent password in the build like we can set the remote server.  I am trying to move away from using JumpDesktop for support.

**Comment by SergeyMy** [comment]
since you are creating assemblies from source code, it is easy to add a password generation API
:)
+ look at this code.... there is already something there!
src/core_main.rs
...
--get-id
--password
